vendor/engines/secure_rooms/app/services/secure_rooms/access_handlers/occupancy_handler.rb

```diff
@@ -47,6 +47,13 @@ def new_occupancy
         )
       end
 
+      def complete_existing_occupancy!
+        existing_occupancy.mark_orphaned!
+        if existing_occupancy.order_completable?
+          existing_occupancy.order_detail.complete!
```

Comment:
I forget, does this update the `order_status` as well as the state? We should probably include a spec for that.